PubNub/Network/Parsers/PNSubscribeParser.m

```diff
@@ -274,25 +254,28 @@ + (NSMutableDictionary *)messageFromData:(id)data
 
 + (NSMutableDictionary *)presenceFromData:(NSDictionary *)data {
     
+#warning ask about how this is affected
+    NSDictionary *payload = data[@"d"];
```

Comment:
@jzucker2 We have _kPNMessagePayloadDataKey_ key, why _"d"_ is used here directly? To make life simpler. you can pass objects stored under _kPNMessagePayloadDataKey_ key right into this method (same with _-messageFromData:withAdditionalParserData:_) as _data_ and you don't have to change or pull our data inside of those methods.